packages/chakra-ui-core/src/CReset/tests/CReset.test.js

```diff
@@ -0,0 +1,47 @@
+import CReset from '../'
```

Comment:
Is there a reason you decided to go with `@vue/test-utils`? For testing we're mostly relying on using `@testing-library/vue` like I had earlier mentioned [in this comment](https://github.com/chakra-ui/chakra-ui-vue/pull/287#issuecomment-686874639).

I initially used VTU but we decided in the middle of the project to use the Testing Library because it has a more concise syntax and provides more utilities out of the box.

Reply:
For your reference, please see this:

https://testing-library.com/docs/vue-testing-library/intro

Reply:
I also do realize that we haven't yet added a testing guide for the community. So these kinds of problems might arise. So I will go ahead and create an issue for it.

Reply:
`@vue/test-utils:mount` provides a wrapped Vue component that helps testing the component code directly. `@testing-library/vue:render` utilizes `@vue/test-utils:mount` but doesn't return the wrapped component only the rendered DOM element.

CReset provides no DOM element. So, I found no way to test its functionality outside the component itself.

Reply:
@freality we avoid testing component code directly this is also called as `testing implementation details` read more about it: https://kentcdodds.com/blog/testing-implementation-details/

`CReset` provides no dom element you are right but it injects styles. So we just test the injected styles 🙂